bin/runtime-common/src/messages.rs

```diff
@@ -449,6 +449,7 @@ pub mod target {
 		WeightCredit: Get<Weight>,
 	{
 		type DispatchPayload = FromBridgedChainMessagePayload<CallOf<ThisChain<B>>>;
+		type DispatchLevelResult = ();
```

Comment:
@svyatonik 
do you want to change this to `type DispatchLevelResult = Result<Outcome, Error>` (I didnt try if compiles) or `()` is ok as default for `FromBridgedChainMessageDispatch`?

Reply:
IMHO `()` is enough for our testnets - dispatch result is not important here, only delivery